samples/DotNet/Microsoft.Azure.ServiceBus/BasicSendReceiveUsingQueueClient/BasicSendReceiveUsingQueueClient.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.ServiceBus" Version="1.0.0" />
+    <PackageReference Include="Microsoft.Azure.ServiceBus" Version="1.0.1" />
```

Comment:
this is a future placeholder assuming that will be the next version iteration.. but will update it to 1.* like you are suggesting..